docs/examples/physics_using_pybullet/README.rst

```diff
@@ -1,4 +1,4 @@
-Physics
+Integrate Physics using pybullet
 -------
```

Comment:
1. `-------` should cover the whole title, otherwise, there is an error
2. In the sentence below the title, Can you replace `Tutorials` by `demos`